scripts/ae_integration_test.sh

```diff
@@ -77,3 +76,10 @@ gcloud container builds submit \
   --substitutions "_DEPLOYED_APP_URL=$DEPLOYED_APP_URL" \
   $dir
 
+# run a cleanup build once tests have finished executing
```

Comment:
Can we do the cleanup only if the version was autogenerated?
Otherwise, we'll lose the benefit of fast deployments to the same running version.

Reply:
Ah yep, good call!

Reply:
You can also just add an argument to specify whether to kill the version.

Reply:
I'm a little wary of adding more flags - I'd prefer to kill the versions only if we're also creating them in this script.